presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergSplitSource.java

```diff
@@ -121,7 +119,7 @@ private ConnectorSplit toIcebergSplit(TupleDomain<IcebergColumnHandle> predicate
         StructLike partition = scanTask.file().partition();
         PartitionSpec spec = scanTask.spec();
         Map<PartitionField, Integer> fieldToIndex = getIdentityPartitions(spec);
-        ImmutableMap.Builder<Integer, String> partitionKeys = ImmutableMap.builder();
+        Map<Integer, String> partitionKeys = new HashMap<>();
```

Comment:
nit: can you use ImmutableMap.Builder so that the code style is consistent with Presto codebase?

Reply:
The Guava Immutable classes can’t contain null values, but HashMap can.